values.yaml

```diff
@@ -61,3 +61,11 @@ tests:
   organization: tf-operator
   # moduleSource defaults to this repository.
   # moduleSource: git::https://github.com/hashicorp/terraform-helm.git//test/module
+
+# If the TFE instance the operator talks to is using a private CA then CACerts
+# can be used to install a custom CA bundle in the operator's container that 
+# will allow it to communicate with TFE.
+#
+# CACerts can be set like this:
+# helm install <other parameters> --set-file CACerts=/path/to/certs.pem
+CACerts: ""
```

Comment:
Not sure about this, but will setting empty string here satisfy the required condition and not show an error when no CA is provided?

Reply:
Yeah, I tested it.